src/vs/workbench/browser/labels.ts

```diff
@@ -514,7 +514,7 @@ class ResourceLabelWidget extends IconLabel {
 		const resource = toResource(this.label);
 		const label = this.label.name;
 
-		if (this.options && typeof this.options.title === 'string') {
+		if (this.options && (typeof this.options.title === 'string' || typeof this.options.title === 'object')) {
```

Comment:
We seem to be loosing the markdown title later here if a decoration is also there:

https://github.com/microsoft/vscode/pull/108403/files#diff-5815caf71eed5559d097ed2b5e8ca250R548

Reply:
I think it is fair to say that if title is markdown then we do not add the decoration.